java/ql/lib/semmle/code/java/security/StaticInitializationVectorQuery.qll

```diff
@@ -3,6 +3,7 @@
 import java
 import semmle.code.java.dataflow.TaintTracking
 import semmle.code.java.dataflow.DataFlow2
```

Comment:
Drive-by change: we can just remove this import and replace `DataFlow2` with `DataFlow` in its only use below.

```suggestion
```